src/main/java/btdex/ui/Main.java

```diff
@@ -511,6 +505,10 @@ public void update() {
 	}
 
 	private void updateUI() {
+		Icon ICON_CONNECTED = i.get(Icons.CONNECTED);
```

Comment:
This would update icons inside the update thread (running continuously). I don't think this is a good idea.

Reply:
I agree, no need to update it